pontoon/base/models.py

```diff
@@ -1082,6 +1082,7 @@ class Repository(models.Model):
         choices=TYPE_CHOICES
     )
     url = models.CharField("URL", max_length=2000)
+    branch = models.CharField("Branch", default='master', max_length=2000)
```

Comment:
Also, we could retrieve a name of default branch in the first pull and save it to the database, that would safe us from the unecessary "ifs" in the rest of code.